cves/kernel/CVE-2021-3609.yml

```diff
@@ -270,8 +285,10 @@ sandbox:
     Answer should be true or false
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: true
+  note: |
+    I don't know for sure, but the vulnerability did allow users to be elevated
+    to root priveleges, so I assume that there was some sandbox feature violated.
```

Comment:
I would look into this more. I believe you are correct, and the kernel-exploitation blog post gives a lot of helpful information on this.